search.go

```diff
@@ -486,14 +486,14 @@ func (ss *SearchStatus) Merge(other *SearchStatus) {
 // A SearchResult describes the results of executing
 // a SearchRequest.
 type SearchResult struct {
```

Comment:
Would you add commentary above this construct explaining what each of the response attribute stands for. This would then include what our definition for `Cost` is - the user can benefit from this with godocs.